### DISCUSSION TITLE
Alternative screen font

### DISCUSSION THREAD

**Discussion by Trinitek** [cc-tweaked/CC-Tweaked#1163] [General]
What would it take to allow the use of a monospaced font specifically for use on screens? It appears now that it is using the default Minecraft font and snapping it to a grid, which causes kerning issues with narrow letters like `l` and `i`.

I discovered https://github.com/IdreesInc/Monocraft which I think would be a good fit. While it advertises support for ligatures, I am not necessarily interested in that.

**Comment by Lupus590** [comment]
You can set the font that computercraft uses in a resource pack.

**Comment by SquidDev** [comment]
> It appears now that it is using the default Minecraft font and snapping it to a grid, 

[We actually use a custom font](https://github.com/cc-tweaked/CC-Tweaked/blob/mc-1.16.x/src/main/resources/assets/computercraft/textures/gui/term_font.png), so it should be fairly easy to patch that.

**Reply by Trinitek** [reply]
Custom resource pack works, thank you (https://github.com/Trinitek/MonocraftForCCTweaked). Would there be any interest in making that the default?

**Reply by SquidDev** [reply]
Probably not I'm afraid. I'm definitely open to tweaking the font, but don't want to wholesale replace it.

**Reply by Trinitek** [reply]
There ended up being not very many changes to the letters; only `i` `l` and `t` were changed, which makes the biggest difference. Mostly punctuation was affected with the biggest visual difference for `:` `;` `{` `}` and `~`, which might be awkward if you're used to looking at the Minecraft default.

So what about just tweaking `i` `l` and `t`?

**Reply by SquidDev** [reply]
I can't say for sure, but definitely happy to look at a PR if you're willing to open one!
